lib/util.js

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+// module API
+const api = {};
+module.exports = api;
```

Comment:
This should be using ESM, not CommonJS.

Reply:
Fixed in 3fa9ac619ab107f961dfdf65b462e5baf7b07f78.